exchanges/okgroup/okgroup.go

```diff
@@ -632,11 +630,17 @@ func (o *OKGroup) SendHTTPRequest(ep exchange.URL, httpMethod, requestType, requ
 		Verbose:       o.Verbose,
 		HTTPDebugging: o.HTTPDebugging,
 		HTTPRecording: o.HTTPRecording,
+	}
+
+	err = o.SendPayload(ctx, request.Unset, func() (*request.Item, error) {
+		return item, nil
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. OKGroup has unimplemented rate limits (not in scope, but relevant)